iaas/iaas.go

```diff
@@ -42,4 +14,5 @@ type Machine struct {
 	Name      string `json:"name"`
 	Kind      string `json:"kind"`
 	SSHKeysID []int  `json:"ssh_keys_id"`
+	CertsDir  string `json:certs_dir`
```

Comment:
struct field tag `json:certs_dir` not compatible with reflect.StructTag.Get: bad syntax for struct tag value